src/olympia/blocklist/tasks.py

```diff
@@ -229,6 +231,10 @@ def upload_filter(generation_time, is_base=True, upload_stash=False):
     with storage.open(mlbf.filter_path, 'rb') as filter_file:
         attachment = ('filter.bin', filter_file, 'application/octet-stream')
         server.publish_attachment(data, attachment)
+        statsd.incr('blocklist.tasks.upload_filter.upload_mlbf')
+    statsd.incr(
+        'blocklist.tasks.upload_filter.upload_mlbf.'
+        f'{"base" if is_base else "full"}')
```

Comment:
Why is this "base" and "full"? Those names seem confusing to me. Would "base" and "update" be better?

Reply:
It's the same as the values of the constants BLOCKLIST_RECORD_MLBF_BASE an BLOCKLIST_RECORD_MLBF_UPDATE above in the file.  Rob wanted the attachment type constants to be "base and "full" (full, as in it's a full bloomfilter rather than a stash file, would would be the update) and I've use the same for consistency with the metadata in remote settings.